models/spring-ai-ollama/src/test/java/org/springframework/ai/ollama/BaseOllamaIT.java

```diff
@@ -46,9 +44,8 @@ public static OllamaApi buildOllamaApiWithModel(String model) {
 	}
 
 	public static void ensureModelIsPresent(OllamaApi ollamaApi, String model) {
-		logger.info("Start pulling the '{}' model. The operation can take several minutes...", model);
-		ollamaApi.pullModel(new OllamaApi.PullModelRequest(model));
-		logger.info("Completed pulling the '{}' model", model);
+		var ollamaModelManager = new OllamaModelManager(ollamaApi);
+		ollamaModelManager.pullModel(model, PullModelStrategy.WHEN_MISSING);
```

Comment:
The integration test setup now uses the same auto-pull functionality as the production code.